config/initializers/deface.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
```

Comment:
No need to have this file, let the main rails app handle deface. 
Deface is also present in extra_user_fields and bunch of other decidim gems.